db/schema.rb

```diff
@@ -786,6 +786,13 @@
     t.index ["icn"], name: "index_mobile_users_on_icn", unique: true
   end
 
+  create_table "nod_notifications", force: :cascade do |t|
+    t.text "payload_ciphertext"
+    t.text "encrypted_kms_key"
+    t.datetime "created_at", precision: 6, null: false
+    t.datetime "updated_at", precision: 6, null: false
```

Comment:
What's `precision: 6` do?

Reply:
That sets the number of significant digits (which is a default of 6 for datetime).